client/db/types.go

```diff
@@ -420,20 +424,18 @@ func DecodeBalance(b []byte) (*Balance, error) {
 }
 
 func decodeBalance_v0(pushes [][]byte) (*Balance, error) {
-	if len(pushes) < 2 {
-		return nil, fmt.Errorf("decodeBalances_v0: expected >= 2 pushes. got %d", len(pushes))
-	}
-	if len(pushes)%2 != 0 {
-		return nil, fmt.Errorf("decodeBalances_v0: expected an even number of pushes, got %d", len(pushes))
+	if len(pushes) != 3 {
+		return nil, fmt.Errorf("decodeBalances_v0: expected 3 pushes. got %d", len(pushes))
```

Comment:
Mentioned in matrix chat, but it is really desirable to not change the db scheme at this point, unless we want to version it now and write and upgrade.  It seems like the trackedTrades give us the contractlocked amount.

But regarding the balance in the DB, I'm not sure why we really need it stored, except perhaps to show a sensible last known balance on before connecting the wallets.

Reply:
Since the contractlocked amount can be computed by core, I agree we can do without storing that balance.

Now that I read this comment though, I realize that the initial computation when loading the wallet from db happens (or may happen) before the trades are loaded from db, the former is done on client launch, the latter on login. Just have to make sure the contractlocked amounts are updated after loading the trades from db.